Packs/RubrikPolaris/Integrations/RubrikPolaris/RubrikPolaris_test.py

```diff
@@ -2350,3 +2350,311 @@ def test_gps_vm_recover_files_command_success(client, requests_mock, empty_respo
+        remove_empty_elements(response_data.get('outputs_when_not_whitelisted'))
+    assert response.outputs.get(f'{OUTPUT_PREFIX["PAGE_TOKEN_USER_ACCESS"]}(val.name == obj.name)') == \
+           remove_empty_elements(response_data.get('page_token_2'))
+    assert response.readable_output == hr_data
+
+
+@pytest.mark.parametrize("args, error", [
+    ({"limit": "0"}, ERROR_MESSAGES['INVALID_LIMIT'].format(0)),
+    ({"limit": MAXIMUM_PAGINATION_LIMIT + 1}, ERROR_MESSAGES['INVALID_LIMIT'].format(MAXIMUM_PAGINATION_LIMIT + 1)),
+    ({"sort_order": "INC"}, ERROR_MESSAGES['INVALID_SORT_ORDER'].format("INC")),
+])
+def test_rubrik_sonar_user_access_list_command_with_invalid_args(client, args, error):
+    """
+    Test case scenario for invalid arguments for rubrik_sonar_user_access_list_command.
+
+    Given:
+        -args: Contains arguments for the command.
+    When:
+        -Invalid value is passed in arguments
+    Then:
+        -Raises ValueError and asserts error message
+    """
+    from RubrikPolaris import rubrik_sonar_user_access_list_command
+
+    with pytest.raises(ValueError) as e:
+        rubrik_sonar_user_access_list_command(client, args=args)
+    assert str(e.value) == error
+
+
+@pytest.mark.parametrize("empty_response", [True, False])
+def test_rubrik_sonar_user_access_get_command_success(client, requests_mock, empty_response):
+    """
+    Test case scenario for rubrik_sonar_user_access_get_command with valid case.
+
+    When:
+        - Calling rubrik_sonar_user_access_get_command.
+    Then:
+        - Verifies mock response with actual response.
+    """
+    from RubrikPolaris import rubrik_sonar_user_access_get_command
+
+    # Load test data
+    response_data = util_load_json(os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                                                "test_data/sonar_user_access_get_response.json"))
+    with open(os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                           "test_data/sonar_user_access_get_response_hr.md"), 'r') as f:
+        hr_data = f.read()
+
+    args = {"user_id": "S-1-0-01-0000000000-0000000000-000000000-0001", "include_whitelisted_results": True}
+    if empty_response:
+        requests_mock.post(BASE_URL_GRAPHQL, [{"json": response_data.get('empty_response')}])
+        response = rubrik_sonar_user_access_get_command(client, args=args)
+        assert response.readable_output == MESSAGES["NO_RESPONSE"]
+    else:
+        requests_mock.post(BASE_URL_GRAPHQL, [{"json": response_data.get('raw_response')}])
+        response = rubrik_sonar_user_access_get_command(client, args=args)
+
+        assert response.raw_response == response_data.get('raw_response')
+        assert response.outputs.get(f'{OUTPUT_PREFIX["USER_ACCESS"]}(val.principalId == obj.principalId)') == \
+            remove_empty_elements(response_data.get('outputs'))
```

Comment:
same here, let's split to different tests if needed.

Reply:
Done